### PR TITLE
Send notification when reg agent creates a declaration

### PR DESCRIPTION
**Pull request by Nil20**

### CODE REVIEW

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]
@Nil20 It would be better to not merge the events from workflow, instead let's send it towards notification service & in there we can use the same handler if needed.

File: packages/workflow/src/records/handler/create.ts
Comment (Zangetsu101) [inline packages/workflow/src/records/handler/create.ts:292]: @Nil20 Could remove this marriage event check please? This is not needed anymore as the `isNotificationEnabled` function handles the marriage event now

File: packages/workflow/src/records/handler/create.ts
Comment (Zangetsu101) [inline packages/workflow/src/records/handler/create.ts:292]: We don't want to disable notification for 'sent-for-approval' no?

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: packages/workflow/src/records/handler/create.ts
Comment (Nil20) [reply packages/workflow/src/records/handler/create.ts:292]: Thought i fixed it in a previous commit!

**Review by Zangetsu101 — APPROVED** [review]

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**
